### PR TITLE
Update Safari TP to release 12

### PR DESCRIPTION
**Pull request by chicoxyzzy**

### CODE REVIEW

File: data-es5.js [hunk 1]
```diff
@@ -121,16 +121,16 @@ exports.browsers = {
...
     family: 'JavaScriptCore',
     short: 'SF TP',
     unstable: true
   },
   webkit: {
-    full: 'Webkit r204466 (August 15, 2016)',
+    full: 'Webkit r204559 (August 17, 2016)',
     family: 'JavaScriptCore',
     short: 'WebKit',
-    obsolete: false
```
Comment (chicoxyzzy) [inline data-es5.js]: O_o

**Comment by afmenez** [comment]
LGTM

**Comment by chicoxyzzy** [comment]
New Sierra beta was released yesterday. Checked out results for new Safari 10 build and they are identical to earlier version.

File: data-es5.js [hunk 2]
```diff
@@ -121,16 +121,16 @@ exports.browsers = {
...
     short: 'SF TP',
     unstable: true
   },
   webkit: {
-    full: 'Webkit r204466 (August 15, 2016)',
+    full: 'WebKit r204915 (August 24, 2016)',
     family: 'JavaScriptCore',
     short: 'WebKit',
-    obsolete: false
+    unstable: true
```
Comment (chicoxyzzy) [inline data-es5.js:133]: notice this ^

**Comment by chicoxyzzy** [comment]
updated to Safari TP release 12

**Pull request merged**